### DISCUSSION TITLE
Prisma is in the error log wrong typed

### DISCUSSION THREAD

**Discussion by DanielOberlechner** [prisma/prisma#14581] [Q&A] [answered]
`% pip install -U prsima       
ERROR: Could not find a version that satisfies the requirement prsima (from versions: none)
ERROR: No matching distribution found for prsima
`

Google showed me that it is wrong typed because of this error I'm getting.
Hope you guys can find the string and correct the word "prisma".

Kind regards,
Daniel Oberlechner

**Comment by austincrim** [comment, marked as answer]
Hey Daniel!

Is that the output of running `pip install -U prsima`? If so, the errors shows the misspelling because "prisma" was misspelled in the install command.

**Reply by DanielOberlechner** [reply]
Upps sorry for wasting your time. Should have looked better.

Kinda a python issue with the whole package management. I hope they come up with a better solution the next time.

Thank you for helping out!

Kind regards,
Daniel Obelchner

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
